Move connectivty tutorials (#2282)
giantswarm/docs: src/content/tutorials/connectivity/ingress/service-type-loadbalancer/index.md

  - How do I configure an ingress controller behind an ELB that terminates SSL?
  - How do I configure an internal Load Balancer on AWS?
  - How do I configure an internal Load Balancer on Azure?
  - How do I configure an internal Load Balancer on GCP?
last_review_date: 2024-08-26
aliases:
  - /advanced/connectivity/ingress/service-type-loadbalancer
  - /guides/services-of-type-loadbalancer-and-multiple-ingress-controllers/
  - /advanced/ingress/service-type-loadbalancer-multi-ic/
owner:
  - https://github.com/orgs/giantswarm/teams/team-cabbage
---

Next to using the default ingress nginx controller, on cloud providers (currently AWS and Azure), you can expose services directly outside your cluster by using services of type `LoadBalancer`.

You can use this to [expose single services](#service-of-type-lb) to the internet. It's also possible, to [install additional ingress nginx controllers]({{< relref "/tutorials/connectivity/ingress/multi-nginx-ic" >}}) to expose a subset of your services with a different ingress controller configuration.

__Note__: that this functionality can't be used on premises in most of the occasions.

> 
> ## Exposing a single Service {#service-of-type-lb}
> 
> Setting the `type` field of your service to `LoadBalancer` will result in your service being exposed by a dynamically provisioned load balancer.
> 
> You can do this with any service within your cluster, including services that expose several ports.
> 
> The actual creation of the load balancer happens asynchronously, and information about the provisioned balancer will be published in the Service’s `status.loadBalancer` field, like following:
> 

Author: ```suggestion

## Exposing a single service {#service-of-type-lb}

Setting your service `type` field to `LoadBalancer` will expose it to a dynamically provisioned load balancer.

You can do this with any service within your cluster, including services that expose several ports.

The actual creation of the load balancer happens asynchronously, and information about the provisioned balancer will be published in the Service’s `status.loadBalancer` field, like the following:

```
